api/handlers/products.js

```diff
@@ -0,0 +1,276 @@
+const Pool = require('pg').Pool;
+const pool = new Pool({
+    user: 'masteruser',
+    password: "ecommdbpw",
+    host: 'ecommdb.covnxnivrrc5.ap-southeast-2.rds.amazonaws.com',
+    database: "ecommdb",
+    port: 5432,
+    ssl: true
+})
+module.exports.getProducts = async (event) => {
+    
+try{
+   
+    const category = event.queryStringParameters && event.queryStringParameters.category
+
+    if(category){
+        const result = await pool.query('SELECT * FROM products WHERE category= $1', [category])
+        return {
+            headers: {
+                'Access-Control-Allow-Origin': '*',
+               
+              },
+            statusCode: 200,
+            body: JSON.stringify(result.rows)
+        }
+    } else{
+       
+            const result = await pool.query('SELECT * FROM products ORDER BY id ASC')
+            return {
+                headers: {
+                    'Access-Control-Allow-Origin': '*',
+                    
+                  },
+                statusCode: 200,
+                body: JSON.stringify(result.rows)
+            }
+    }
+
+} catch(error){
+    return {
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+           
+          },
+        statusCode: 500,
+        body: JSON.stringify({error: 'Internal server error'})
+    }
+
+}
+     
+}
+//get categories
+module.exports.getCategories = async (event) => {
+   
+    try{
+        const result =  await pool.query('SELECT DISTINCT category FROM products');
```

Comment:
Would it be better to have the categories return in alphabetical order? Right now they are being returned in random order.